firmware/application/apps/ui_sigfrx.hpp

```diff
@@ -50,6 +51,11 @@ class SIGFRXView : public View {
     uint8_t last_channel;
     uint8_t detect_counter = 0;
 
+    RxRadioState radio_state_{
+        1750000 /* bandwidth */,
+        868110000 /* sampling rate */
```

Comment:
Looks quite high for a valid sampling rate, no ?

Reply:
After looking at it in ui_navigation:
  { "SIGFOX", 	ui::Color::dark_grey(),	&bitmap_icon_fox,		[&nav](){ nav.push<NotImplementedView>(); } }, // SIGFRXView
  So well, we may fix it but it may not add anything X-D

Reply:
Done. For an unused app, meh.

Reply:
Thanks for fixing it.